onnx/onnx-data_pb.h

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) ONNX Project Contributors.
```

Comment:
is this file needed?

Reply:
Removing this file is not trivial -- perhaps something to investigate in a follow-on PR. onnx-operators.proto has a similar onnx-operators_pb.h as well.